src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -174,7 +204,7 @@ export default withToolbar(MediaOverview, function() {
                 onClick: () => {
                     this.clearDatagrids();
                     router.restore(
-                        COLLECTION_ROUTE,
+                        USER_SETTING_PREFIX,
```

Comment:
I would keep the `MEDIA_ROUTE` and `USER_SETTING_PREFIX` constants separate. it feels strange to route to a prefix...